extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -1338,6 +1337,16 @@ public static enum ApplicationType {
     @ConfigItem
     public Optional<Provider> provider = Optional.empty();
 
+    /**
+     * Indirectly verify that the opaque (binary) access token is valid by using it to request UserInfo.
+     * Opaque access token is considered valid if the provider accepted this token and returned a valid UserInfo.
+     * You should only enable this option if the opaque bearer access tokens have to be accepted but OpenId Connect
+     * provider does not have a token introspection endpoint.
+     * This property will have no effect when JWT tokens have to be verified.
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean verifyOpaqueAccessTokenWithUserInfo;
```

Comment:
Can we move it to some more specific group, it is seem to be very specific to be at the top group level, how about `Tokens` group ? `Authentication` group is reserved for the code flow related properties

Reply:
You are right, fixed.